polygon/bridge/reader.go

```diff
@@ -149,7 +165,6 @@ func (r *RemoteReader) EventTxnLookup(ctx context.Context, borTxHash libcommon.H
 	return reply.BlockNumber, reply.Present, nil
 }
 
-// Close implements bridge.ReaderService. It's a noop as there is no attached store.
```

Comment:
Why is this removed?

Reply:
I thought it is an unnecessary comment after the bridge.ReaderService interface got removed. If someone removes this function they will get a compilation error in the consuming packages that expect to see this Close function on a bridge reader. However I'll add back the "no-op as there is no attached store" comment bit back as I agree it is useful to have.